docs/howto/add_activable_element_from_selection.md

```diff
@@ -0,0 +1,72 @@
+# How-to: Add activable element in a selection form
+
+The Hybrid Platform UI application provides a component called `ez-selection-activate-element` which
+is an HTML custom element. This component gives the possibility to easily implement activable
```

Comment:
> This component gives the possibility to easily implement activable
> elements (like buttons) that will be enabled and can triggers actions only if some elements in a
> list are selected.
> The selection is done with the help of checkboxes or radio buttons. Some activable elements
> need at least one or more element checked to be enabled, but others will need one and only one
> element checked to be enabled.

can be simplified to something like:

*This component gives the ability to enable or disable an HTML element (typically a button) depending on a selection made with checkboxes or radio button.*